scripts/release.ts

```diff
@@ -21,19 +21,32 @@ const verboseExec = (cmd: string) => {
 };
 
 const checkPrerequisites = () => {
+  const minHubVersion = "2.14.1";
+  let result;
+
   try {
-    execSync("git --version", {
-      stdio: "ignore",
-    });
-    execSync("hub --version", {
-      stdio: "ignore",
-    });
+    result = execSync("hub --version", {
+      stdio: "pipe",
+    }).toString("utf-8");
   } catch {
-    throw new Error(`
-  Please install missing dependencies:
-    - https://git-scm.com
-    - https://github.com/github/hub
-`);
+    console.log(
+      `\nCould not find the GitHub CLI tool >= ${minHubVersion}.\n` +
+        "You can get it here:\n\n" +
+        "  https://github.com/github/hub\n"
+    );
```

Comment:
You could throw an exception instead but this is also fine.

Reply:
I prefer it this way, to me the error message stands out clearer without the exception trace and it signals that this is how the program is intended to behave. I left the exceptions for when something unexpected happens.

See https://github.com/radicle-dev/radicle-upstream/pull/1601#discussion_r574492916 for how the error output looks vs how it looks like when there's an actual exception:

```
$ scripts/release.ts
(node:10256) UnhandledPromiseRejectionWarning: Error: Couldn't parse the version number
    at checkPrerequisites (/Users/rudolfs/work/radicle-upstream-2/scripts/release.ts:46:13)
    at main (/Users/rudolfs/work/radicle-upstream-2/scripts/release.ts:250:3)
    at Object.<anonymous> (/Users/rudolfs/work/radicle-upstream-2/scripts/release.ts:287:1)
    at Module._compile (internal/modules/cjs/loader.js:1201:30)
    at Module.m._compile (/Users/rudolfs/work/radicle-upstream-2/node_modules/ts-node/src/index.ts:1056:23)
    at Module._extensions..js (internal/modules/cjs/loader.js:1221:10)
    at Object.require.extensions.<computed> [as .ts] (/Users/rudolfs/work/radicle-upstream-2/node_modules/ts-node/src/index.ts:1059:12)
    at Module.load (internal/modules/cjs/loader.js:1050:32)
    at Function.Module._load (internal/modules/cjs/loader.js:938:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:10256) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:10256) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
✨  Done in 1.90s.
```